src/middlewared/middlewared/plugins/sysctl/sysctl_info_linux.py

```diff
@@ -0,0 +1,66 @@
+import errno
+import os
+
+from middlewared.service import CallError, Service
+from middlewared.utils import run
+
+from .sysctl_info_base import SysctlInfoBase
+
+
+ZFS_MODULE_PARAMS_PATH = '/sys/module/zfs/parameters'
+
+
+class SysctlService(Service, SysctlInfoBase):
+
+    def get_value(self, sysctl_name):
+        raise NotImplementedError
+
+    def not_found_error(self, name):
```

Comment:
Why a method for something used only once?

Reply:
I think it will be useful moving on and I was using it in multiple places before as well but then changed my code to use something different. However, if you think we should not have this, I can make the required change : )